src/spikeinterface/extractors/tests/test_nwbextractors_streaming.py

```diff
@@ -254,6 +254,45 @@ def test_sorting_s3_nwb_remfile(tmp_path):
     check_sortings_equal(reloaded_sorting, sorting)
 
 
+@pytest.mark.streaming_extractors
+@pytest.mark.skip("Too slow")
+def test_sorting_s3_nwb_zarr(tmp_path):
+    file_path = (
```

Comment:
Where are these? As in who ise the owner of those files?

Reply:
The are in a public bucket, so tests should run fine!

Reply:
I was just curios if they were yours or where did you find them.

Reply:
yes, they are from a workshop we made in May with the Allen Institute :)